BallBounceGame/Assets/GameStart/GameScript.cs

```diff
@@ -19,6 +19,9 @@ public class GameScript : MonoBehaviour
     SpriteRenderer m_SpriteRenderer;
 
     Text gameText;
+    float gameTimer = 0;
+    float finishTime = 0;
+    int ball_count = 0;
```

Comment:
nice mix of camel case and snek case 😂😂😂😂

Reply:
Naming conventions are hard